src/mumble/Settings.cpp

```diff
@@ -644,6 +645,7 @@ void Settings::load(QSettings* settings_ptr) {
 	SAVELOAD(iMaxImageWidth, "net/maximagewidth");
 	SAVELOAD(iMaxImageHeight, "net/maximageheight");
 	SAVELOAD(qsServicePrefix, "net/serviceprefix");
+	SAVELOAD(iMaxInFlightTCPPings, "net/maxinflighttcppings");
```

Comment:
Yes, I actually wanted to implement something like this.

How about treating `"net/maxinflighttcppings" <= 0` as disabling the feature?